lib/utilities/mouseDownDrag.js

```diff
@@ -1,49 +1,72 @@
 /**
+ * Callback which is called upon at some point during a drag event in React.
+ *
+ * @callback drawCallback
+ * @param {event} event The original mouse event.
+ */
+
+/**
```

Comment:
This is a hint to JSDoc that the various callback methods defined below have a specific interface/set of requirements (ie. they take an event parameter).